geometry_msgs/PointStamped[] personVector
